{
  "title": "CVE-2021-32613",
  "cve": "CVE-2021-32613",
  "notes": {
    "description": "In radare2 through 5.3.0 there is a double free vulnerability in the pyc parse via a crafted file which can lead to DoS.",
    "published": "2021-05-14",
    "modified": "2021-07-29"
  },
  "references": [
    {
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/Y3S7JB46PONPHXZHIMR2XDPLGJCN5ZIX/",
      "description": "FEDORA:FEDORA-2021-834f900f53"
    },
    {
      "url": "https://lists.fedoraproject.org/archives/list/package-announce@lists.fedoraproject.org/message/W3LPB5VGCIA7WA55FSB3YZQFUGZKWD7O/",
      "description": "FEDORA:FEDORA-2021-ca59eb65a9"
    },
    {
      "url": "https://github.com/radareorg/radare2/commit/5e16e2d1c9fe245e4c17005d779fde91ec0b9c05",
      "description": "MISC:https://github.com/radareorg/radare2/commit/5e16e2d1c9fe245e4c17005d779fde91ec0b9c05"
    },
    {
      "url": "https://github.com/radareorg/radare2/commit/a07dedb804a82bc01c07072861942dd80c6b6d62",
      "description": "MISC:https://github.com/radareorg/radare2/commit/a07dedb804a82bc01c07072861942dd80c6b6d62"
    },
    {
      "url": "https://github.com/radareorg/radare2/issues/18666",
      "description": "MISC:https://github.com/radareorg/radare2/issues/18666"
    },
    {
      "url": "https://github.com/radareorg/radare2/issues/18667",
      "description": "MISC:https://github.com/radareorg/radare2/issues/18667"
    },
    {
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1959939",
      "description": "MISC:https://bugzilla.redhat.com/show_bug.cgi?id=1959939"
    },
    {
      "url": "https://github.com/radareorg/radare2/issues/18679",
      "description": "MISC:https://github.com/radareorg/radare2/issues/18679"
    }
  ]
}
